platforms/core-configuration/base-diagnostics/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id("gradlebuild.distribution.api-java")
+    id("gradlebuild.instrumented-java-project")
+}
+
+description = """Base diagnostic reporting infrastructure and reports that are applicable to all Gradle builds.
+These reports and types do not require dependency management to be present."""
+
+dependencies {
+    api(projects.baseServices)
+    api(projects.core)
+    api(projects.coreApi)
+    api(projects.enterpriseLogging)
+    api(projects.fileCollections)
+    api(projects.jvmServices) // TODO: shouldn't be here
```

Comment:
Maybe improve this comment a bit as to _why_ this is here now if it shouldnt be?

Reply:
Added.